ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -150,6 +150,7 @@ if pull_request:
         ' --ros-version ' + str(ros_version) +
         (' --run-abichecker' if run_abichecker else '') +
         (' --require-gpu-support' if require_gpu_support else '') +
+        (' --custom-rosdep-update-options="' + ' '.join(custom_rosdep_update_options) + '"' if custom_rosdep_update_options else '') +
```

Comment:
We should consider changing the argument type for this to use `nargs=argparse.REMAINDER` to avoid the need to wrap the value in quotes. The approach here is fragile and will break if the arguments need to contain quotes or other characters meaningful to the shell.

I'm not entirely sure if we should block this PR on that, and I'm not sure if it might be a breaking change to do so, but as someone who is clearly utilizing this feature already, I'm hoping you can tell me!